blogs/2022-06-28-tips-kafka-streams-distributed.md

```diff
@@ -59,64 +59,64 @@ All join provided by Kafka Streams were designed thinking with aggregation in mi
 
 Now, let's handle the data stored in Kafka. The main logic is a [Kafka Stream](https://github.com/kestra-io/kestra/blob/develop/runner-kafka/src/main/java/io/kestra/runner/kafka/streams/WorkerInstanceTransformer.java) that:
-- Create a global `KTable` with all the `WorkerInstance`
-- On every change, it will listen to `WorkerInstance` changed
-- If there is a new `WorkerInstance`, we look at the partition assigned on this one, if there is an overlap between this instance partitions and the previous, we are sure that the previous `WorkerInstance` is dead (in Kafka, you can't have 2 consumers on the same partition).
-- We only need to look at the Task affected to this `WorkerInstance` and resend them for processing.
+- Creates a global `KTable` with all the `WorkerInstance.`
+- Listens to `WorkerInstance` changed for every change.
+- If there is a new `WorkerInstance,` we look at the partition assigned. Where there is an overlap between this instance's partitions and the previous ones, we are sure that the previous `WorkerInstance` is dead. This is because you can't have two consumers on the same partition in Kafka. 
+- We only need to look at the task affected for this `WorkerInstance` and resend it for processing.
 
-Et voila 🎉 We have detection of dead consumers for free only with Kafka  API.
+Et voila 🎉 We can detect dead consumers for free with Kafka API.
 
 ## State store `all()`
 
-We need to use a [GlobalKTable](https://kafka.apache.org/31/documentation/streams/developer-guide/dsl-api.html#streams_concepts_globalktable) in order to detect [flow trigger](/docs/developer-guide/triggers/flow.html): for all the flow on the cluster, we test all the flows [conditions](/docs/developer-guide/conditions/) in order to find matching flows. For this, we are using an API to fetch all flows from a `GlobalKTable` using `store.all()` that returned all the flows, fetching RockDb.
+We need to use a [GlobalKTable](https://kafka.apache.org/31/documentation/streams/developer-guide/dsl-api.html#streams_concepts_globalktable) to detect [flow trigger](/docs/developer-guide/triggers/flow.html). To find matching flows, we test all the flows on the cluster [conditions](/docs/developer-guide/conditions/). For this purpose, we are using an API to fetch all flows from a `GlobalKTable` using `store.all()` that returned all the flows, fetching RockDb.
 
-The first assumption is that `all()` returned is an object (Flow in our case), as the API return Object, but we discovered that the `all()` will:
-- fetch all the data from RockDB (ok for that)
-- deserialize the data from RockDB that is stored as byte and map it to concrete Java POJO
+The first assumption is that `all()` returned is an object (Flow in our case), as the API return Object. However, we discovered that the `all()` will:
+- Fetch all the data from RockDB (But for what?).
```

Comment:
Maybe we need to say more on the question in brackets so that it makes sense to the reader.